Changelog.md

```diff
@@ -8,6 +8,7 @@ Compiler Features:
 
 
 Bugfixes:
+ * AST: Export ``canonicalName`` for ``UserDefinedValueTypeDefinition`` and ``ContractDefinition``.
```

Comment:
Not sure if this should be part of bugfix. Also might need a rebase.

Reply:
Why not? We export `canonicalName` for all other TypeDefinition AST nodes.

Reply:
Alright. Fine for me; actually both "Bugfixes" and "AST changes" would work for me.